README.md

```diff
@@ -1,21 +1,21 @@
 #
 
-<div align="center">
+<p align="center">
   <img src="./frontend/public/android-chrome-512x512.png" width="8%">
-  <h1>
+  <h1 align="center">
     MUMBLE
   </h1>
-</div>
+<p align="center">An Open Source Social Media Platform and Public Forum for Questions and Discussions, built for Developers.</p>
+</p>
 
-An Open Source Social Media Platform and Public Forum for Questions and Discussions, built for Developers.
 
-## 🔗 Links
+## 🔗 Links :
 
 - **[🔴 Live Demo](https://www.mumble.dev/)**
 - **[🎨 UI Kit Docs/Preview](http://mumble-lp.s3-website-us-west-2.amazonaws.com/)**
-- **[✅ Discord Server](https://discord.gg/TxgpyK8pzf)**
+- **[✅ Join The Discord Server](https://discord.gg/TxgpyK8pzf)**
 
-## Download & Setup Instructions
+## Download & Setup Instructions :
```

Comment:
I'm not a big fan of adding these colons : to the end of all the titles.  I don't know of any other readme on github that does this.  Checkout https://github.com/facebook/react/blob/master/README.md or https://github.com/angular/angular/blob/master/README.md